doc/sphinx-guides/source/developers/remote-users.rst

```diff
@@ -55,15 +55,32 @@ Now load the configuration defined in ``oidc-keycloak-auth-provider.json`` into
 
 You should see the new provider, called "OIDC-Keycloak", under "Other options" on the Log In page.
 
-You should be able to log into Keycloak with the following credentials:
-
-- username: kcuser
-- password: kcpassword
+You should be able to log into Keycloak with the one of the following credentials:
```

Comment:
Nice list of users and groups. The Groups are not mapped to dataverse groups, right?  Could we add a short sentence stating, that there is yet no mapping of roles/groups between OIDC and keycloack?

Reply:
Nope, not supported yet, just preparing. Please feel free to leave a suggestion.

Reply:
Please leave a suggestion if you want this to be changed. I'm fine with changing it, we can re-add when we have better OIDC mapping support.

Reply:
Ok, I'm leaving a ```suggestion to drop the Group column.

Reply:
Thanks, added! Also created a new issue #9969